wordpress-mobile/WordPress-iOS
# Add Create Site button to Domain selection view

Fixes #8415 

**To test:**
On the Domain Suggestions page:
- The button view is hidden initially.
- When a domain is selected
    - A checkmark appears in the row.
    - The button view appears at the bottom.
- When a new search is initiated, the button view is hidden.

**As designed:**
![create_site_button](https://user-images.githubusercontent.com/1816888/34841402-6c722006-f6c5-11e7-9f90-8f0cc3289545.png)

**Implemented:**
![implemented](https://user-images.githubusercontent.com/1816888/34841449-9158f732-f6c5-11e7-9eee-be249d21ffa3.png)
